packages/picasso/src/Popper/test.tsx

```diff
@@ -48,7 +48,7 @@ const defaultPopperProps = {
   style: {
     width: '300px'
   },
-  popperOptions: getPopperOptions({})
+  popperOptions: getPopperOptions({}, false)
```

Comment:
We can add a default value (`false`) to the second argument of `getPopperOptions` and leave this file unchanged.

```suggestion
  popperOptions: getPopperOptions({})
```

Reply:
https://github.com/toptal/picasso/pull/2340/commits/7e8c09b21f95e7e9f01448b70fe7240923395734